.github/workflows/checking.yml

```diff
@@ -0,0 +1,13 @@
+name: Link Checking
+
+on:
+  schedule:
+    - cron: 0 0 * * * # At 00:00 (UTC)
+
+jobs:
+  link-checking:
+    name: Link Checking
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: JSCIG/link-checker-action@v1
```

Comment:
This should be a hash, not a tag or branch.

Reply:
tag based is the general practice of github action

Reply:
That makes no difference to me. If we need enhancements in the future, we can change the hash.

Reply:
for any github action, use hash based?

Reply:
That is my preference, yes. I can make exceptions for GitHub-owned actions (github.com/actions/*).

Reply:
i choose general community practice

unless i have a must be reason to this practice. (e.q: write in https://github.com/tc39/how-we-work)

Reply:
@septs this isn't a "PR author choose" kind of thing; there's security/reliability concerns.

Reply:
This issue need a guideline.

@ljharb and @michaelficarra can write to how-we-work ?

Reply:
i want to see tc39 github actions practice guideline.

Reply:
The guideline is "whatever the repo maintainer chooses", generally - I'm not sure why this is a sticking point.